WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs

```diff
@@ -99,6 +99,47 @@ public async Task TrezorTKataAsync()
 			Assert.Equal(TransactionCheckResult.Success, checkResult);
 		}
 
+		[Fact]
+		public async Task TrezorTKata2Async()
+		{
+			// --- USER INTERACTIONS ---
+			// Initialize your Trezor T with:
+			// > more maid moon upgrade layer alter marine screen benefit way cover alcohol
+			// Run this test.
+			// signtx request: confirm
+			//
+			// --- USER INTERACTIONS ---
+
+			var network = Network.Main;
+			var client = new HwiClient(network);
+			using var cts = new CancellationTokenSource(ReasonableRequestTimeout);
+			var enumerate = await client.EnumerateAsync(cts.Token);
+			HwiEnumerateEntry entry = Assert.Single(enumerate);
+			Assert.NotNull(entry.Path);
+			Assert.Equal(HardwareWalletModels.Trezor_T, entry.Model);
+			Assert.True(entry.Fingerprint.HasValue);
+			Assert.Null(entry.Code);
+			Assert.Null(entry.Error);
+			Assert.Null(entry.SerialNumber);
+			Assert.False(entry.NeedsPassphraseSent);
+			Assert.False(entry.NeedsPinSent);
+
+			string devicePath = entry.Path;
+			HardwareWalletModels deviceType = entry.Model;
+			HDFingerprint fingerprint = entry.Fingerprint.Value;
```

Comment:
Unused?

Reply:
Add Assert and check if the returned value is correct.